script.js

```diff
@@ -18,4 +18,175 @@ document.documentElement.addEventListener('click', (e) => {
     overlay.classList.remove('overlay-show');
     document.body.classList.remove('stop-scrolling');
   }
+});
+
+// popup window setup
+// Project Cards Info
+const projectData = [
+  {
+    image: 'asset/snapshot1.svg',
+    title: 'Tonic',
+    technologies: ['Back End Dev', '2015'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript'],
+  },
+  {
+    image: 'asset/snapshot2.svg',
+    title: 'Multi-Post Stories',
+    technologies: ['Back End Dev', '2015'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript'],
+  },
+  {
+    image: 'asset/snapshot3.svg',
+    title: 'Tonic',
+    technologies: ['Back End Dev', '2015'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript'],
+  },
+  {
+    image: 'asset/snapshot4.svg',
+    title: 'Multi-Post Stories',
+    technologies: ['Back End Dev', '2015'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript'],
+  },
+];
+
+// Render project cards
+// const projectCard = document.querySelector('.card__container');
+
+// function displayCard(id) {
+//   projectCard.innerHTML += `
+//     <div class="items card-1">
+//       <div>
+//         <img src="${projectData[id].image}" alt="screenshot" class="screenshot">
+//       </div>
+//       <div class="card__description">
+//         <div>
+//           <h3 class="card-title">${projectData[id].title}</h3>
+//           <div class="card-bg-info">
+//             <p>Canopy</p>
+//             <ul>
+//                 <li>${projectData[id].technologies[0]}</li>
+//                 <li>${projectData[id].technologies[1]}</li>
+//             </ul>
+//           </div>
+//         </div>
+//         <div class="card__text">
+//           <p>
+//           ${projectData[id].text}
+//           </p>
+//         </div>
+//         <div class="card__skills">
+//           <ul>
+//             <li>${projectData[id].skills[0]}</li>
+//             <li>${projectData[id].skills[1]}</li>
+//             <li>${projectData[id].skills[2]}</li>
+//           </ul>
+//         </div>
+//         <div>
+//           <button class="btn">See Project</button>
+//         </div>
+//       </div>
+//     </div>
+//   `;
+// }
+
+// for (let i = 0; i < projectData.length; i += 1) {
+//   // displayCard(i);
```

Comment:
- Kindly ensure all commented codes to make your codebase more readable 🤝🏽

Reply:
@rotimiazeez Please, suggest if I have to delete the comments or should I keep them?